docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       context: ./tests/api
       dockerfile: Dockerfile
     environment:
-      WX_NOW_TIMESTAMP: ${WX_NOW_TIMESTAMP}
+      WX_NOW_TIMESTAMP: ${WX_NOW_TIMESTAMP:-}
```

Comment:
when we specify a blank default, this silences `WX_NOW_TIMESTAMP variable is not set` warnings